articles/sentinel/process-events-normalization-schema.md

```diff
@@ -200,14 +201,14 @@ The process event schema references the following entities, which are central to
 |<a name="targetuserid"></a> **TargetUserId**   | Recommended | String     | A unique ID of the target user. The specific ID depends on the system generating the event. For more information, see [The User entity](normalization.md#the-user-entity).            <br><br> Example: `S-1-5-18`    |
 | **TargetUserIdType**               | Recommended | String     | The type of the user ID stored in the [TargetUserId](#targetuserid) field. For more information, see [The User entity](normalization.md#the-user-entity).            <br><br> Example:  `SID`  |
 | **TargetUserSessionId**            | Optional     | String     |The unique ID of the target user's login session. <br><br>Example: `999`          <br><br>**Note**: The type is defined as *string* to support varying systems, but on Windows this value must be numeric. <br><br>If you are using a Windows or Linux machine and used a different type, make sure to convert the values. For example, if you used a hexadecimal value, convert it to a decimal value.     |
-| <a name="targetprocessname"></a>**TargetProcessName**              | Mandatory    | string     |The file name of the target process image file.   <br><br>     Example:  `C:\Windows\explorer.exe`     |
-| **TargetProcessCompany**       | Optional     | String     |The name of the company that created the target process image file.   <br><br>   Example:  `Microsoft` |
+| <a name="targetprocessname"></a>**TargetProcessName**              | Mandatory    | string     |The name of the target process, commonly derived from the image file or executable file used to define the initial code and data mapped into the process' virtual address space.   <br><br>     Example:  `C:\Windows\explorer.exe`     |
```

Comment:
`The name of the target process. This name is commonly derived from the image or executable file that's used to define the initial code and data that's mapped into the process' virtual address space.`